app/views/sign_in_tokens/email_not_recognised.html.erb

```diff
@@ -6,19 +6,20 @@
       <%= t('page_titles.email_not_recognised') %>
     </h1>
     <p class="govuk-body">
-      We did not recognise the email address you entered.
-      You may have mistyped it, or you may not have access.
+      If you mistyped your email address, <%= govuk_link_to 'try to sign in again', sign_in_path %>.
```

Comment:
Not sure if you saw my last comment I left about this line, but I think we should play back the email address they supplied otherwise there's no way for them to double check what was submitted. They maybe a registered user but just had a typo in the email address they supplied . so by playing it back it gives them the opportunity to check why it didn't allow them access.

Perhaps something along the lines of "The email address you supplied was x, if this is incorrect please try again"( with a link back to the sign in page. It also shows empathy by not not explicitly calling the user out and blaming them for supplying the wrong email address.